charts/headlamp/README.md

```diff
@@ -0,0 +1,66 @@
+# headlamp
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.4.0](https://img.shields.io/badge/AppVersion-0.4.0-informational?style=flat-square)
+
+Headlamp is an easy-to-use and extensible Kubernetes web UI.
+
+**Homepage:** <https://github.com/kinvolk/headlamp/tree/main/deploy/helm>
```

Comment:
I think even after merging. This will be 404. It should be `<https://github.com/kinvolk/headlamp/tree/main/charts/headlamp>`